docs/core/install/runtime.md

```diff
@@ -37,7 +37,32 @@ macOS has standalone installers that can be used to install the .NET Core 3.1 ru
 
 ## Install with a package manager
 
-You can install the .NET Core Runtime with many of the common Linux package managers. For more information, see [Linux Package Manager - Install .NET Core](linux-package-manager-rhel7.md).
+You can install the .NET Core Runtime with many of the common Linux package managers. For more information, see [Linux Package Manager - Install .NET Core](linux-package-managers.md).
+
+Installing with a package manager is only supported for 64-bit CPUs. If you're installing the .NET Core Runtime with a 32-bit CPU, follow the [Download and manually install](#download-and-manually-install) instructions below.
```

Comment:
> 32-bit CPU

This is one example, but I'm pretty sure arm/arm64 are much more popular.

In fact... it might be misleading to say "64-bit CPU" and "32-bit CPU", since it's really the x86-64 *arch* that's supported, and I think processors that support x86-64 also generally support installing a 32-bit x86 OS (i686?). The category also would suggest we support arm64, a "64-bit CPU", but we don't.

I'd suggest replacing 64-bit with x86-64 or x64, and use "ARM or ARM64" as the example "other" type.